src/module.ts

```diff
@@ -97,6 +97,11 @@ export const plugin = getPanelPluginOrFallback(
             showIf: (config: HeatmapFieldConfig) =>
               config.colorPalette !== 'custom' && config.colorPalette !== 'fieldOptions',
           })
+          .addColorPicker({
```

Comment:
We need to check in which version of Grafana the color picker option was added. Currently, the plugin can run on >=7.0.3, but we might need to bump the required minimum required version depending on when this was introduced.